x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/find.ts

```diff
@@ -0,0 +1,576 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { asyncForEach } from '@kbn/std';
+import { UserAtSpaceScenarios } from '../../../../scenarios';
+import { getTestRuleData, getUrlPrefix, ObjectRemover } from '../../../../../common/lib';
+import { FtrProviderContext } from '../../../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function findBackfillTests({ getService }: FtrProviderContext) {
```

Comment:
I may have missed it, but didn't see a FT for pagination of the find.  Feels like we should have one.

Reply:
Added in https://github.com/elastic/kibana/pull/179975/commits/91a9408f4f0f8176da3da5b873c8272252bd9f21